### DISCUSSION TITLE
Calculate loss using ddf in 3D image registration without labels

### DISCUSSION THREAD

**Discussion by nehagoyal1994** [Project-MONAI/MONAI#2976] [Q&A]
Hi All,

I am experimenting 3D image registration without the help of labels. To calculate the loss, I am currently using MSELoss().
 I want to use the ddf in the loass calculation after each epoch; for example :
                                    
                                    loss   =   image_loss(pred_image, fixed_image)   * \                                                           
                                                            regularization(ddf)                        
 
    where               image_loss = MSELoss()  and  regularization = BendingEnergyLoss()   

Please See:    https://deepreg.readthedocs.io/en/latest/tutorial/registration.html#unsupervised

So far the loss is not exponentially decreasing for 200 epochs.
Could you please guide me with loss calculation?

Thanks in advance

**Comment by Nic-Ma** [comment]
Hi @kate-sann5100 @wyli ,

Could you please help share some comments here as you are experts for registration?

Thanks in advance.

**Comment by wyli** [comment]
the `image_loss * regularization` looks suspicous to me, perhaps that should be `image_loss + lambda * regularization`?

**Reply by nehagoyal1994** [reply]
I already tried with lambda  ( different values )  but each time the loss was increasing exponentially rather than decreasing
